style.css

```diff
@@ -7,11 +7,30 @@
 body {
     font-family: 'Poppins', sans-serif;
 }
+
+.logo{
+    position: fixed;
+    top: 0;
+    left: 0;
+}
 .navbar {

```

Comment:
The Cart icon is not showing please fix that too

Reply:
will make the necessaryy fixes